modules/shared/components/molecules/PostFooterCreation/PostFooterCreation.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import type { IPostFooterCreation } from './IPostFooterCreation';
+import styles from './PostFooterCreation.module.css';
+import Privacy from '../Privacy/Privacy';
```

Comment:
I know the team agreed to name the component Privacy but we have a duplication (the word avatar) here ../Privacy/Privacy

when we started the project we agreed to name it [Privacy/index.tsx]
(In the future if we need to make any change we had to discuss why and what are pros and cons of that)

Why you changed it and what is the argument, benefits.
(by you I mean all members 😂😂😂 not only @ahmedelsayad11  )